test/js/color-handler.js

```diff
@@ -15,10 +15,10 @@ suite('color-handler', function() {
   });
   test('color interpolation', function() {
     assert.equal(webAnimations1.propertyInterpolation('color', '#00aa11', '#aa00bb')(0.2), 'rgba(34,136,51,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0), 'transparent');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0.5), 'rgba(0,68,136,0.500)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(2), 'rgba(0,255,0,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(-1), 'rgba(255,0,0,1)');
+    assert.equal(webAnimations1.propertyInterpolation('fill', 'transparent', '#004488')(0), 'transparent');
```

Comment:
I think we need to use something other than 0 for the progress on this test, to distinguish successful interpolation from a 50-50 flip.